src/pytest_postgresql/compat.py

```diff
@@ -1,19 +1,23 @@
 """pscypog2 Compatibility module."""
-from typing import Any
+from typing import Any, TYPE_CHECKING
 from platform import python_implementation
 
 
+__all__ = ("psycopg2", "cursor", "connection", "check_for_psycopg2")
+
+
 try:
     import psycopg2
 except ImportError:
     psycopg2 = False
 
 # pylint:disable=unused-import
 if not psycopg2:
-    # if there's no postgres, just go with the flow.
-    # pylint:disable=invalid-name
-    cursor = Any
-    connection = Any
+    if not TYPE_CHECKING:
+        # if there's no postgres, just go with the flow.
+        # pylint:disable=invalid-name
```

Comment:
oh... we can rid of pylint comment here

Reply:
Are we intentionally not using pylint? Should I (separately) delete all such comments or fix it so pylint passes?

Reply:
intentionally for now. 
I left the formatting and most of the checks to black. As it often happened that one breaks the other and it takes a bit of tinkering with configuration for them to cooperate.

Reply:
If it's ok, I'd prefer to delete the unused ignores in a separate PR.

Reply:
nah, leave it be then